### PR TITLE
Clean up before release

### PR DESCRIPTION
**Pull request by pablocarmu**
## Description

PR to fix https://github.com/owncloud/ios-app/issues/247

It adds a new boolean environment variable called `oc:app.is-beta-build`. This should be activated and coded to true when the build is going to be a beta build. 

This new environment variable is used to select messages between different build types.

Also, it adds new mail for feedback and a new link in the _recommend to a friend_ section

## Related Issue

https://github.com/owncloud/ios-app/issues/247

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ownCloud/Tools/VendorServices.swift [hunk 1]
```diff
@@ -45,6 +46,18 @@ class VendorServices : NSObject {
 		return ""
 	}
 
+	var isBetaBuild: Bool {
+		if let isBetaBuild = self.classSetting(forOCClassSettingsKey: .isBetaBuild) as? Bool {
+			if isBetaBuild {
```
Comment (mneuwert) [inline ownCloud/Tools/VendorServices.swift]: @pablocarmu I think you could just `return isBetaBuild` here, no if condition needed

File: ownCloud/Tools/VendorServices.swift [hunk 2]
```diff
@@ -53,13 +66,20 @@ class VendorServices : NSObject {
 	func recommendToFriend(from viewController: UIViewController) {
 		let message = """
 <p>I want to invite you to use ownCloud on your smartphone!</p>
-<a href="https://itunes.apple.com/app/owncloud/id543672169?mt=8">Download here</a>
+<a href="https://itunes.apple.com/app/id1359583808?mt=8">Download here</a>
```
Comment (mneuwert) [inline ownCloud/Tools/VendorServices.swift]: @pablocarmu we could make an AppStore link and also E-Mail addresses configurable trough build settings rather then hard-coding them. What if a customer would like to release branded AppStore version? cc: @michaelstingl what do you think?

Comment (pablocarmu) [reply ownCloud/Tools/VendorServices.swift]: I can code this using the `OCClassSettings` mechanism, so we can provide default email & AppStore link that could be overwritten by an MDM or config file.

File: ownCloud/Tools/VendorServices.swift [hunk 3]
```diff
@@ -53,13 +66,20 @@ class VendorServices : NSObject {
 	func recommendToFriend(from viewController: UIViewController) {
 		let message = """
 <p>I want to invite you to use ownCloud on your smartphone!</p>
```
Comment (michaelstingl) [inline ownCloud/Tools/VendorServices.swift]:
```suggestion
<p>I want to invite you to use $BRANDNAME on your smartphone!</p>
```

File: ownCloud/Tools/VendorServices.swift (hunk 2 above)
Comment (michaelstingl) [reply ownCloud/Tools/VendorServices.swift]: Yes, we need a boolean switch to disable it, and another parameter (URL?) that can be customised.

Comment (pablocarmu) [reply ownCloud/Tools/VendorServices.swift]: These are the new variables I've coded:

- **Variables**:
	- app-store-link : Points to the app's link in the app store.
		- type: String
		- default: `https://itunes.apple.com/app/id1359583808?mt=8`
	- feedback-email : Email to send the feedback mail.
		- type: String
		- default: `ios-app@owncloud.com`
	- recommend-to-friend-enabled : Option to send en email with the App Store link.
		- type: Bool
		- default: `true`
	- send-feedback-enabled : Send an email to feedback-email with some feedback.
		- type: Bool
		-default: `true`

**Comment by jesmrec** [comment]
Checks in variables:

- [X] app-store-link. Default value OK. Changed value OK
- [X] feedback-email : Default value OK. Changed value OK
- [X] recommend-to-friend-enabled : true OK, false OK
- [X] send-feedback-enabled : true OK, false OK

**Comment by jesmrec** [comment]
Works for me. Approved.

**Pull request merged**